### ISSUE TITLE
SyntaxError: invalid property id...

### ISSUE THREAD

**Issue by pantsofpie** [folke/polydock#2]
Having an issue trying to run Polydock. I have ensured all the dependencies are installed, and have tried both from the current folder (bin/polydock) and installed to ~/.local/bin/

I have also tried with and without the default configuration installed to ~/.config/polydock/

This is the result every time:

```
# bin/polydock

(gjs:29578): Gjs-CRITICAL **: 15:44:55.103: JS ERROR: SyntaxError: invalid property id @ bin/polydock:138
Script bin/polydock threw an exception
```

**Comment by folke** [comment]
That's odd.

Can you verify that the line 138 is something like:
```js
 this.settings = { ...defaults };
```

Can you give me the output of
```shell
$ gjs --version
```

Thanks!

**Comment by pantsofpie** [comment]
Line 138 looks correct:

```
134     __init() {this.verbose = true;}
135     __init2() {this.themePath = [];}
136 
137     constructor() {Config.prototype.__init.call(this);Config.prototype.__init2.call(this);
138       this.settings = { ...defaults };
139       this.path = GLib.path_get_dirname(
140         realpath(imports.system.programInvocationName)
141       );
142       this.path = resolve(GLib.path_get_dirname(this.path));
143       this.themePath.push(`${this.path}/config/themes`);
144       this.theme = resolve(`${this.path}/config/themes/default.css`);
145     }
```
And the gjs version:
```
# gjs --version
gjs 1.52.5
```

This was the current version in the Ubuntu 18.04 (Linux Mint 19.3) repos.

**Comment by folke** [comment]
Thanks. I think I know what's the problem here.

In Fedora, the current version is 1.64.3, which probably contains some additional JS language features.

Will see to transpile / downlevel so it also works on older versions.

I'll keep you updated!

**Comment by folke** [comment]
I just released [v2.0.2](https://github.com/folke/polydock/releases/tag/2.0.2)

All the code now gets automatically transpiled to ES5.

This hopefully fixes the syntax error in your case.

**Comment by pantsofpie** [comment]
Can confirm v2.0.2 fixes it! :)

Thanks!
